test/e2e/performanceprofile/functests/utils/node_inspector/inspector.go

```diff
@@ -19,58 +19,91 @@ import (
 	testutils "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils"
 	testclient "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/client"
 	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/daemonset"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/images"
 	testlog "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/log"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/namespaces"
 	testpods "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/pods"
 )
 
 const serviceAccountSuffix = "sa"
 const clusterRoleSuffix = "cr"
 const clusterRoleBindingSuffix = "crb"
 
-func Create(cli client.Client, namespace, name, image string) error {
-	serviceAccountName := fmt.Sprintf("%s-%s", name, serviceAccountSuffix)
-	sa := createServiceAccount(serviceAccountName, namespace)
-	if err := cli.Create(context.Background(), sa); err != nil && !errors.IsAlreadyExists(err) {
+var initialized bool
+var namespace *corev1.Namespace = namespaces.NodeInspectorNamespace
+var nodeInspectorName = testutils.NodeInspectorName
+
+// initialize would be used to lazy initialize the node inspector
+func initialize() error {
+	if initialized {
+		return nil
+	}
+	// Create the test namespace
+	err := testclient.DataPlaneClient.Create(context.TODO(), namespace)
+	if err != nil && !errors.IsAlreadyExists(err) {
```

Comment:
We should handle `errors.IsAlreadyExists(err) == true` separately, otherwise we will continue to `create()` and will return with error there